tests/testthat/test-metadata.R

```diff
@@ -560,49 +650,57 @@ test_that("xportr_metadata: Check metadata interaction with other functions", {
     rlang::set_names(tolower)
 
```

Comment:
This will make the messages dissappear and you can remove the other `suppressMessages`

Otherwise, the `suppressMessages` that were removed from test will appear when running the tests locally

Win-win?

```suggestion

skip_if_not_installed("withr")
withr::local_message_sink(withr::local_tempfile())
```

![image](https://github.com/atorus-research/xportr/assets/211358/56afc2e7-aecc-40a5-9a76-5318af717b5f)

Reply:
I think this is great, those messages have always annoyed me. I made a new issue (#223) referencing this and the magrittr requirement lines